scripts/deploy-operator-crd-scaling.sh

```diff
@@ -19,4 +19,5 @@ oc wait deployment new-pro-controller-manager -n "$TNF_EXAMPLE_CNF_NAMESPACE" --
 
 make addrole
 kubectl apply -f config/samples --validate=false
+sleep 10
```

Comment:
This might work for now but I'd rather go with a busy-wait like in these examples: https://github.com/kubernetes/kubernetes/issues/83242#issuecomment-1048715671
Can be done in further PR, though.

What about adding another sleep/busy-wait before the "oc wait" on line 18?